R/write.magpie.R

```diff
@@ -22,7 +22,7 @@
 #' data column. In the case that more than one year and data column is supplied
 #' several files are written with the structure filename_year_datacolumn.asc
 #'
-#' @param x MAgPIE-object
+#' @param x magclass or RasterBrick object. RasterBrick works only with file_types "nc", "asc", "grd" and "tif".
```

Comment:
It should be highlighted that magclass is the default here. Perhaps write something like:

"a magclass object. An exception is that formats written via the raster package (currently "nc", "asc", "grd" and "tif") also accept RasterBrick objects which have been previously created from a magclass object via as.RasterBrick)."